frontend/src/components/Form.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './form.css';
+import PropTypes from 'prop-types';
+
+function Form({ username, setUserName, userName, isLogin }) {
+  return (
+    <form>
+      <div className="username">
+        <label htmlFor="username-name" className="username-font">
+          Username
+          <input
+            className={username ? 'username-box-used' : 'username-box'}
+            type="username"
+            label="Create Username"
+            value={userName}
+            onChange={(e) => setUserName(e.target.value)}
+            required
+            placeholder="Username"
+          />
+        </label>
+
+        {isLogin ? (
+          <p className={username ? 'username-used' : 'username-not-used'}>
```

Comment:
Let's store the result of this ternary in a variable so that it can be reused on line 27